pkg/package/modfile.go

```diff
@@ -193,6 +193,7 @@ type Dependency struct {
 	FullName string `json:"-" toml:"full_name,omitempty"`
 	Version  string `json:"-" toml:"version,omitempty"`
 	Sum      string `json:"-" toml:"sum,omitempty"`
+	Package  string `json:"package" toml:"package,omitempty"`
```

Comment:
At present, we do not need to serialize json when the output of this field `Package`, do not output first, to prevent users from relying on it, causing difficulties for our subsequent changes and updates. So set `json:"-"`